app/views/dashboard/index.html.erb

```diff
@@ -1 +1,5 @@
 <h1><%= @merchant.name %></h1>
+<nav class="nav">
+  <%= link_to 'My Items', "/merchant/#{@merchant.id}/items"%>
```

Comment:
One thing to consider -- we should be able to use the path helpers here instead of the raw path string

Reply:
Ohhh good point!! Thanks @aschwartz1